### PR TITLE
chore: add license

### PR DESCRIPTION
**Pull request by davidlj95**
**What is the purpose of this pull request?**

- [ ] Documentation update
- [ ] Bug fix
- [ ] New rule
- [ ] Changes an existing rule
- [ ] Add autofixing to a rule
- [x] Other, please explain:

Update LICENSE according to license specified in `package.json`

ISC one (as specified in package.json) from GitHub template
https://github.com/licenses/license-templates/blob/master/templates/isc.txt

**What changes did you make? (Give an overview)**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Copyright (c) 2020, xjamundx
```
Comment (iamdarshshah) [inline LICENSE:1]:
```suggestion
Copyright (c) 2016, xjamundx
```
Please review [this](https://github.com/xjamundx/eslint-plugin-promise#license). I believe it should be 2016, @xjamundx Correct me if I'm wrong and don't know if we should keep your `{{github_username}}` as the `{{ organization }}` or it's better to add your name instead i.e., `Jamund Ferguson`.

**Comment by iamdarshshah** [comment]
@davidlj95 Thanks for the PR. 🎉 

 Let's wait for @xjamundx 's comment on whether the details mentioned in the license are correct or it still needs any change.

File: LICENSE (hunk 1 above)
Comment (xjamundx) [reply LICENSE:1]: I think it should be my name, yes, but I can probably do this.

Comment (iamdarshshah) [reply LICENSE:1]: @xjamundx Awesome, thanks for the update.

**Pull request merged**